config/routes.rb

```diff
@@ -5,6 +5,8 @@
   namespace :slack do
     post "event"
   end
+  
+  post "/slack/:command", to: "slack#command"
```

Comment:
Can we move this into the `namespace :slack` block above and perhaps just call `post "command"` otherwise as you have it now we're gonna have problems if one of the commands is ever named "event".

Reply:
I tried but it didn't work, I can look into it further.